homeassistant/components/device_tracker/bluetooth_le_tracker.py

```diff
@@ -65,11 +74,13 @@ def discover_ble_devices():
     # to 0
     for device in load_config(yaml_path, hass, 0):
         # check if device is a valid bluetooth device
-        if device.mac and device.mac[:3].upper() == BLE_PREFIX:
+        if device.mac and device.mac[:4].upper() == BLE_PREFIX:
             if device.track:
-                devs_to_track.append(device.mac[3:])
+                _LOGGER.debug("Adding %s to BLE tracker", device.mac)
+                devs_to_track.append(device.mac[4:])
             else:
-                devs_donot_track.append(device.mac[3:])
+                _LOGGER.debug("Adding %s to BLE do not track", device.mac)
+                devs_donot_track.append(device.mac[4:])
```

Comment:
Why only the first 4 bytes? You'll probably end up matching all devices from a manufacturer if they have the same OUI